packages/core/src/lib/tooltip/__tests__/tooltip.spec.svelte

```diff
@@ -3,11 +3,13 @@ import { Tooltip, type FloatingPlacement, type TooltipVisibility } from '$lib';
 
 export let location: FloatingPlacement = 'top';
 export let state: TooltipVisibility | undefined = undefined;
+export let hoverDelayMs: number | undefined = undefined;
```

Comment:
[Nit] Not sure how much we want to stick to our acronym casing convention, here, but maybe `hoverDelayMS`